crates/subspace-farmer-components/src/plotting.rs

```diff
@@ -21,19 +21,15 @@ use tracing::info;
 /// Defines retry policy on error during piece acquiring.
 #[derive(PartialEq, Eq, Clone, Debug, Copy)]
 pub enum PieceGetterRetryPolicy {
-    /// Exit on the first error
-    NoRetry,
-
-    /// Try N times
+    /// Retry N times (including zero)
```

Comment:
I would prefer changing `Limited` option argument to `NonZero` type instead and keeping `NoRetry` option as a more explicit API.

Reply:
I was thinking about that. While I'm not strongly opposed, it feels a bit too cumbersome to use, so I wasn't sure it is worth it. It doesn't feel like 0 retries is fundamentally nonsensical, it felt quite natural that 0 retries means no retries and from that reasoning I decided we probably don't need a duplicate `NoRetry` anymore.